src/components/LoginPage/LoginPageComponent.js

```diff
@@ -89,7 +89,9 @@ const Button = css`
 
 const GoogleLoginButton = styled.button`
   ${Button}
-  float: right;
+  position: absolute;
+  right: 2rem;
+  top: 1rem;
```

Comment:
Did we have any problems with styles on login page?
@TomaTomoe pls review

Reply:
It adds a little space on top. Generally using I would not recommend using float, instead you can use flex container as complete replacement, or other solutions. In our case we need position button on the right upper corner and it will look nicer with a little margin (space) between edge of the page and button itself.